pom.xml

```diff
@@ -62,22 +60,22 @@
             <version>${jackson.version}</version>
         </dependency>
 
-        <dependency>
+        <dependency> <!-- TODO: Remove GSON in favor of Jackson -->
```

Comment:
We'll remove it once we're not using it anymore; the note is just there as a reminder.